.php-cs-fixer.dist.php

```diff
@@ -62,12 +62,21 @@
         'numeric_literal_separator' => true,
         'phpdoc_order' => [
             'order' => [
+                'deprecated',
+                'final',
+                'readonly',
+                'internal',
+                'no-named-arguments',
+                'import-type',
                 'type',
```

Comment:
the imported type could be used inside one of the new type aliases, so I would import first, and the define additional type aliases.